js/src/lib/interfaces/Libraries.ts

```diff
@@ -233,14 +233,24 @@ model = timm.create_model("hf_hub:${model.id}", pretrained=True)`;
 const sklearn = (model: ModelData) => {
 	if (model.tags?.includes("skops")) {
 		const skopsmodelFile = model.config?.sklearn?.filename;
+		const skopssaveFormat = model.config?.sklearn?.model_format;
+		if (skopssaveFormat == "pickle")
+		{
+			return `import joblib
+from skops.hub_utils import download
+
+download("${model.id}", "path_to_folder")
```

Comment:
Should this have a security warning as the one we had before? ("if model is a pickle file, make sure it's from a source you trust")

Reply:
SGTM, but also wanted to ask @adrinjalali about it. WDYT?

Reply:
yeah that sounds good to me. We can also have a link to our docs: https://skops.readthedocs.io/en/stable/persistence.html